cli/src/main.rs

```diff
@@ -60,6 +60,10 @@ struct Args {
     #[argh(switch)]
     keep_coordinate_space: bool,
 
+    /// enable the cfg only mode in the dot backend
```

Comment:
This comment gets turned into the `--help` text from the CLI, so let's try to make it a bit more explicit. Suggest:

> In dot output, include only the control flow graph